_news/2021-07-18-cobot-woco-interview.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Works Council interview series #1: Cobot"
+tags: works-councils
+---
+
+The following is the first of a planned series of interviews with Works Councils (*Betriebsräte*) in Berlin tech companies, conducted by Berlin TWC's content team.
+
+For the first episode, we are very happy to hear from the recently elected Works Councillor at Cobot, a small Berlin software company.
+
+**What company do you work for and since when does a Works Council exist/are you part of?**
+
+I work for Cobot, which elected its first Works Councillor (me!) in October 2020.
```

Comment:
I find the `"Me"` identifier a bit evasive. I can understand the desire not to state name, given how small the company is, and would suggest adding an explanation for this.

Reply:
Would you suggest to remove the `me!`, and/or just put something in the intro? Do you have a concrete formulation in mind, as in "who would like to stay anonymous"?